# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wxwidgets24/locale.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur wxPythonSrc-2.4.2.4_orig/src/common/intl.cpp wxPythonSrc-2.4.2.4/src/common/intl.cpp
--- wxPythonSrc-2.4.2.4_orig/src/common/intl.cpp	2003-09-22 21:09:45.000000000 +0200
+++ wxPythonSrc-2.4.2.4/src/common/intl.cpp	2005-03-31 17:50:13.032652824 +0200
@@ -274,7 +274,7 @@
     // search first in prefix/fr/LC_MESSAGES, then in prefix/fr and finally in
     // prefix (assuming the language is 'fr')
     searchPath << prefix << wxFILE_SEP_PATH << lang << wxFILE_SEP_PATH
-                         << wxT("LC_MESSAGES") << wxPATH_SEP
+                         << wxT("LC_MESSAGES/wx24") << wxPATH_SEP
                << prefix << wxFILE_SEP_PATH << lang << wxPATH_SEP
                << prefix << wxPATH_SEP;
 
@@ -303,8 +303,8 @@
 #ifdef __UNIX__
     // add some standard ones and the one in the tree where wxWin was installed:
     searchPath
-        << GetAllMsgCatalogSubdirs(wxString(wxGetInstallPrefix()) + wxT("/share/locale"), lang)
-        << GetAllMsgCatalogSubdirs(wxT("/usr/share/locale"), lang)
+        << GetAllMsgCatalogSubdirs(wxString(wxGetInstallPrefix()) + wxT("/share/locale/wx24"), lang)
+        << GetAllMsgCatalogSubdirs(wxT("/usr/share/locale/wx24"), lang)
         << GetAllMsgCatalogSubdirs(wxT("/usr/lib/locale"), lang)
         << GetAllMsgCatalogSubdirs(wxT("/usr/local/share/locale"), lang);
 #endif // __UNIX__
